ksqldb-functional-tests/src/test/resources/query-validation-tests/greatest.json

```
@@ -0,0 +1,178 @@
+{
+  "comments": [
+    "Tests covering the greatest function."
```

nit:
```suggestion
    "Tests covering the GREATEST function."
```
(Same for least.json below.)